boards/STM32F103ZET6-F103ZE-Board.md

```diff
@@ -308,40 +309,48 @@ title: "STM32F103ZET6 - F103ZE board"
 
 ## TFT
 
-| Pin   | Connected to |
-| ----- | ------------ |
-|       | NRST         |
-|       | Ground plane |
-|       | PD9          |
-|       | PD10         |
-|       | PE15         |
-|       | PD8          |
-|       | PE13         |
-|       | PE14         |
-|       | PE11         |
-|       | PE12         |
-|       | PE9          |
-|       | PE10         |
-|       | PE7          |
-|       | PE8          |
-|       | PD0          |
-|       | PD1          |
-|       | PD14         |
-|       | PD15         |
-|       | PD5          |
-|       | PD4          |
-|       | PG12         |
-|       | PG0          |
-|       | PB2          |
-|       | PB1          |
-|       | PF8          |
-|       | PF9          |
-|       | N.C.         |
-|       | N.C.         |
-|       | VBAT         |
-|       | Ground plane |
-|       | +3.3V rail   |
-|       | Ground plane |
+| Pin   | Connected to |Function        |
+| ----- | ------------ |----------------|
+|   2   | NRST         |                |
+|   1   | Ground plane |                |
+|   4   | PD9          | FSMC D14       |
+|   3   | PD10         | FSMC D15       |
+|   6   | PE15         | FSMC D12       |
+|   5   | PD8          | FSMC D13       |
+|   8   | PE13         | FSMC D10       |
+|   7   | PE14         | FSMC D11       |
+|  10   | PE11         | FSMC D8        |
+|   9   | PE12         | FSMC D9        |
+|  12   | PE9          | FSMC D6        |
+|  11   | PE10         | FSMC D7        |
+|  14   | PE7          | FSMC D4        |
+|  13   | PE8          | FSMC D5        |
+|  16   | PD0          | FSMC D2        |
+|  15   | PD1          | FSMC D3        |
+|  18   | PD14         | FSMC D0        |
+|  17   | PD15         | FSMC D1        |
+|  20   | PD5          | FSMC NWE       |
+|  19   | PD4          | FSMC NOE       |
+|  22   | PG12         | FSMC NE4       |
+|  21   | PG0          | FSMC A10       |
+|  24   | PB2          | Touch CS       |
+|  23   | PB1          | Touch CLK      |
+|  26   | PF8          | Touch (MI)SO   |
+|  25   | PF9          | Touch (MO)SI   |
+|  28   | N.C.         | Note 1         |
```

Comment:
It looks like I mistakenly marked this pin as Not Connected (N.C.). However, since it _is_ connected, it should say something else. Maybe something like `Q1` followed by the pin number of the transistor. I'd like to know what you think about this.

Reply:
Oh, and by the way, you can link to your notes by wrapping the text in a link:

`[Note 1](#note-1)`